streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -92,8 +101,49 @@ public void process(Exchange exchange) {
 			return;
 		}
 		if (payload.get("uuid") == null) {
-			log.error(String.format("No uuid column for table %s ignoring payload %s ", table, payload));
-			return;
+			Connection conn = null;
+			Statement stmt = null;
+			final String USER = "root";
+			final String PASS = "debezium";
+			String sqlStmt = null;
+			try {
+				Class.forName("com.mysql.cj.jdbc.Driver");
+				
+				//Connecting to openmrs database
+				conn = DriverManager.getConnection(DB_URL, USER, PASS);
+				
+				//Creating statement
+				stmt = conn.createStatement();
+				
+				String sql = String.format("SELECT uuid FROM %s WHERE %s = %s", config.getParentTable(),
+				    config.getParentForeignKey(), payload.get(config.getChildPrimaryKey().toString()));
+				
+				ResultSet rs = stmt.executeQuery(sql);
+				rs.next();
+				sqlStmt = rs.getString("uuid");
+				rs.close();
+				
+			}
+			catch (Exception exception) {
+				exception.printStackTrace();
+			}
+			finally {
+				//finally block used to close resources
+				try {
+					if (stmt != null)
+						conn.close();
+				}
+				catch (SQLException exception1) {}
+				try {
+					if (conn != null)
+						conn.close();
+				}
+				catch (SQLException exception2) {
+					exception2.printStackTrace();
+				}
```

Comment:
@gitcliff ,you could probably refactor this and put this jdbc related code in a single private method , it will look cleaner i guess